app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/ApplicationModule.kt

```diff
@@ -48,7 +49,8 @@ object ApplicationModule {
                 .addMigrations(
                     MIGRATION_1_2,
                     MIGRATION_2_3,
-                    MIGRATION_3_4
+                    MIGRATION_3_4,
+                    MIGRATION_4_5
                 )
                 .build()
         } catch (e: IllegalStateException) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c7380d74fc32b4ddceef02c8d9a561008c986f4f/app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/ApplicationModule.kt#L56"></span>The caught exception is swallowed. The original exception could be lost.</td>
    </tr>
  </tbody>
</table>